File: java-advent-of-code/src/test/java/advent/twenty_seventeen/Day13PacketScannersTest.java
package advent.twenty_seventeen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day13PacketScannersTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day13PacketScanners("/twenty_seventeen/Day13-PacketScanners-input.txt");

        assertEquals(2604, dp.getPart1Answer().intValue());
        assertEquals(3941460, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day13PacketScanners("/twenty_seventeen/Day13-PacketScanners-example#1.txt");

        assertEquals(24, dp.getPart1Answer().intValue());
        assertEquals(10, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_seventeen/Day13PacketScannersTest.scala
package advent.twenty_seventeen

import org.scalatest.{FlatSpec, Matchers}

class Day13PacketScannersTest  extends FlatSpec with Matchers {
  "2017 Day 13 - Input" should "calculate answers" in {
    val dp = new Day13PacketScanners("/twenty_seventeen/Day13-PacketScanners-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day13PacketScanners("/twenty_seventeen/Day13-PacketScanners-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
